compiler/modules/front/symbols.bal

```diff
@@ -109,3 +115,22 @@ function lookupImportedConst(ModuleSymbols mod, s:ModuleLevelDefn modDefn, strin
     }
     return err:semantic(`${prefix + ":" + varName} is not defined as a public const`, loc=s:locationInDefn(modDefn), functionName=modDefn.name);
 }
+
+function lookupImplicitImport(string prefix) returns Import? {
+    Import? 'import = implicitImports[prefix];
+    if 'import == () {
+        ModuleExportSemtypes? defns = predefinedSubtypes[prefix];
+        if defns != () {
+            bir:ModuleId moduleId = { org: "ballerina", names: ["lang", prefix] };
```

Comment:
We create the moduleId here and also in front.bal which suggests we don't have quite the right structure.

Reply:
I think we need a readonly table containing records with fields for (1) prefix (2) module id and (3) `readonly & map<SemType>`